__tests__/components/Comments/ContainerComment.test.js

```diff
@@ -262,13 +269,15 @@ describe('ContainerComment should run correctly', () => {
     const indexFeed = 0;
     const findCommentAndUpdate = jest.fn();
     const {getByTestId} = render(
-      <ReplyComment
-        countComment={comments2[0].children_counts.comment}
-        indexFeed={indexFeed}
-        findCommentAndUpdate={findCommentAndUpdate}
-        navigateToReplyView={navigateToReplyView}
-        data={comments2[0].latest_children.comment}
-      />
+      <Context.Provider value={{profile: [{}, () => jest.fn()]}}>
+        <ReplyComment
+          countComment={comments2[0].children_counts.comment}
+          indexFeed={indexFeed}
+          findCommentAndUpdate={findCommentAndUpdate}
+          navigateToReplyView={navigateToReplyView}
+          data={comments2[0].latest_children.comment}
+        />
+      </Context.Provider>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same as above, ensure that the `ReplyComment` component can handle the new `profile` prop and that it doesn't break any existing functionality.
```diff
-      <ReplyComment
-        countComment={comments2[0].children_counts.comment}
-        indexFeed={indexFeed}
-        findCommentAndUpdate={findCommentAndUpdate}
-        navigateToReplyView={navigateToReplyView}
-        data={comments2[0].latest_children.comment}
-      />
+      <Context.Provider value={{profile: [{}, () => jest.fn()]}}>
+        <ReplyComment
+          countComment={comments2[0].children_counts.comment}
+          indexFeed={indexFeed}
+          findCommentAndUpdate={findCommentAndUpdate}
+          navigateToReplyView={navigateToReplyView}
+          data={comments2[0].latest_children.comment}
+        />
+      </Context.Provider>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->